{"identifier":{"url":"doc:\/\/RxFireAuth\/documentation\/RxFireAuth\/UserError\/LocalizedError-Implementations","interfaceLanguage":"swift"},"sections":[],"variants":[{"paths":["\/documentation\/rxfireauth\/usererror\/localizederror-implementations"],"traits":[{"interfaceLanguage":"swift"}]}],"hierarchy":{"paths":[["doc:\/\/RxFireAuth\/documentation\/RxFireAuth","doc:\/\/RxFireAuth\/documentation\/RxFireAuth\/UserError"]]},"schemaVersion":{"major":0,"minor":3,"patch":0},"kind":"article","metadata":{"modules":[{"name":"RxFireAuth"}],"role":"collectionGroup","roleHeading":"API Collection","title":"LocalizedError Implementations"},"topicSections":[{"title":"Instance Properties","identifiers":["doc:\/\/RxFireAuth\/documentation\/RxFireAuth\/UserError\/failureReason","doc:\/\/RxFireAuth\/documentation\/RxFireAuth\/UserError\/helpAnchor","doc:\/\/RxFireAuth\/documentation\/RxFireAuth\/UserError\/recoverySuggestion"],"generated":true}],"references":{"doc://RxFireAuth/documentation/RxFireAuth":{"type":"topic","url":"\/documentation\/rxfireauth","abstract":[],"title":"RxFireAuth","kind":"symbol","identifier":"doc:\/\/RxFireAuth\/documentation\/RxFireAuth","role":"collection"},"doc://RxFireAuth/documentation/RxFireAuth/UserError/recoverySuggestion":{"identifier":"doc:\/\/RxFireAuth\/documentation\/RxFireAuth\/UserError\/recoverySuggestion","title":"recoverySuggestion","abstract":[],"url":"\/documentation\/rxfireauth\/usererror\/recoverysuggestion","role":"symbol","kind":"symbol","type":"topic","fragments":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"text":"recoverySuggestion","kind":"identifier"},{"kind":"text","text":": "},{"kind":"typeIdentifier","preciseIdentifier":"s:SS","text":"String"},{"text":"?","kind":"text"}]},"doc://RxFireAuth/documentation/RxFireAuth/UserError":{"type":"topic","url":"\/documentation\/rxfireauth\/usererror","abstract":[{"text":"Errors thrown by  ","type":"text"},{"type":"codeVoice","code":"UserManagerType"},{"text":" implementations.","type":"text"}],"navigatorTitle":[{"text":"UserError","kind":"identifier"}],"title":"UserError","kind":"symbol","fragments":[{"kind":"keyword","text":"enum"},{"text":" ","kind":"text"},{"kind":"identifier","text":"UserError"}],"identifier":"doc:\/\/RxFireAuth\/documentation\/RxFireAuth\/UserError","role":"symbol"},"doc://RxFireAuth/documentation/RxFireAuth/UserError/helpAnchor":{"identifier":"doc:\/\/RxFireAuth\/documentation\/RxFireAuth\/UserError\/helpAnchor","title":"helpAnchor","abstract":[],"url":"\/documentation\/rxfireauth\/usererror\/helpanchor","role":"symbol","kind":"symbol","type":"topic","fragments":[{"text":"var","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"helpAnchor"},{"text":": ","kind":"text"},{"text":"String","kind":"typeIdentifier","preciseIdentifier":"s:SS"},{"text":"?","kind":"text"}]},"doc://RxFireAuth/documentation/RxFireAuth/UserError/failureReason":{"fragments":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"text":"failureReason","kind":"identifier"},{"kind":"text","text":": "},{"kind":"typeIdentifier","preciseIdentifier":"s:SS","text":"String"},{"text":"?","kind":"text"}],"type":"topic","abstract":[],"identifier":"doc:\/\/RxFireAuth\/documentation\/RxFireAuth\/UserError\/failureReason","title":"failureReason","kind":"symbol","url":"\/documentation\/rxfireauth\/usererror\/failurereason","role":"symbol"}}}